src/api/app/jobs/project_log_rotate.rb

```diff
@@ -1,30 +1,21 @@
 class ProjectLogRotate
   def perform
     event_classes = [Event::Package, Event::Project]
+    event_types = event_classes.flat_map(&:descendants).map(&:name)
```

Comment:
Before it did a `.where` on `Event::Package` and `Event::Project` which does the same thing as this but is harder to work with because you now have two lookup queries as opposed to one. So if I want to fetch 10,000 rows then I can just do `limit(10000)`. The same things get executed but this is just easier to work with.